server/sonar-server/src/main/java/org/sonar/server/issue/index/IssueIndexer.java

```diff
@@ -62,8 +71,22 @@ public void indexAll() {
     doIndex(createBulkIndexer(true), 0L, null);
   }
 
-  public void index(String projectUuid) {
-    super.index(lastUpdatedAt -> doIndex(createBulkIndexer(false), lastUpdatedAt, projectUuid));
+  @Override
+  public void indexProject(String projectUuid, Cause cause) {
+    switch (cause) {
+      case PROJECT_CREATION:
+        // nothing to do, issues do not exist at project creation
+        break;
+      case PROJECT_KEY_UPDATE:
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') This case's code block is the same as the block for the case on line 77. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1871)